dw-hardened-ami/generic_packer_template.json.j2

```diff
@@ -6,14 +6,17 @@
     },
     "builders": [{
       "type": "amazon-ebs",
+      {% if 'source_ami_id' in event and event['source_ami_id']|length > 0 %}
+        "source_ami": "{{ event['source_ami_id']}}"",
+      {% endif %}
```

Comment:
The filter should be an else clause if the source ami id is absent should it not?